### DISCUSSION TITLE
Still some incorrect spells in the cumulative fix update.

### DISCUSSION THREAD

**Discussion by Mrunderwonder** [hythloday/VenturePlanSoDMissions#74] [General]
I'm noticing there are still some spell errors included in the cumulative-fix code.
Think some spells were missed from the original discussions. Here's zealvurte's comments. From the issue #17 thread.  

> I did a comparison of this against my own values, and found the following (marked as verified if I've tested for the correct values):
> 
> * 107 is wrong and using an outdated version of the spell; both effects should be `target=0`, effect 1 should be `damageATK=150`, and effect 2 should be `plusDamageTakenATK=50` (Verified)
> * 306 has `modMaxHP`, but should be `modMaxHPATK` according to the data and tooltip
> * 311's second effect has `modMaxHP`, but should be `modMaxHPATK` according to the data and tooltip
> * 319 has `duration=3`, but should be `duration=4`
> * 322's second effect has `modMaxHP`, but should be `modMaxHPATK` according to the data and tooltip
> * 333 has `modDamageDealt`, but should be `plusDamageDealtATK` (Verified)
> * 341's effects are in the wrong order (Verified)
> * 343's effects are in the wrong order (Verified)
> 
> Therefore everything else should be correct as far as I've been able to verify so far.

Some of them are updated but others are still incorrect according to those comments. I'm confused as to how spells 341 and 342 should be coded as it seems what is there now is way off..... Anyone that can point me in the right direction?

**Comment by zealvurte** [comment]
Ignoring my past comments (as I probably had my own inaccuracies at the time) and doing a fresh comparison to [extra-vs-spells.lua (cumulative-fixes)](https://github.com/Divergentcurl/VenturePlanSoDMissions/blob/cumulative-fixes/extra-vs-spells.lua):
- `107`, effect 1 should be `target=0, damageATK=150`, and effect 2 should be `plusDamageTakenATK=50`
- `319`, should be `duration=4`
- `333`, should be `plusDamageDealtATK=40`
- `342`, effect 2 should be `duration=1`
- `349`, should be `damageATK=10`

**Reply by Mrunderwonder** [reply]
Thanks.

**Comment by Divergentcurl** [comment]
Apparently I had somehow overlooked these fixes. I have updated my branch according to @zealvurte's reply. Thanks for noticing.
